.github/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## Release 0.2.0 (development release)
+
```

Comment:
Do you want to add the changelog template here as well (i.e., the headings)? We usually do this in other packages to make it easier to add an entry in under the correct heading.